### PR TITLE
vmanomaly: new version, reflect config updates

### PR DESCRIPTION
**Pull request by zekker6**
- add "bearer_token" parameter for read/write configs
- add "verify_tls" parameter for read/write configs
- add "extra_filters" parameter for read config

### CODE REVIEW

File: hack/vmanomaly-lint-hack.yaml [hunk 1]
```diff
@@ -2,8 +2,17 @@ eula: true
...
+    basicAuth:
+      username: asdf
+      password: asdf
+    extra_filters:
+      - '{cluster!=""}'
+      - '{job!=""}'
   write:
     url: http://cluster-victoria-metrics-cluster-vminsert.default.svc.cluster.local:8480/insert/
+    verify_tls: false
+    bearer_token: asdf
```
Comment (tenmozes) [inline hack/vmanomaly-lint-hack.yaml:15]: should we support file, secret?

Comment (zekker6) [reply hack/vmanomaly-lint-hack.yaml:15]: good point, secret should be an option

Comment (zekker6) [reply hack/vmanomaly-lint-hack.yaml:15]: hm, but atm there is no way to pass either an env variable or a path to the token to vmanomaly
wdyt about adding this in a next version? (after adding this to vmanomaly first)

Comment (tenmozes) [reply hack/vmanomaly-lint-hack.yaml:15]: sure, it works for me

**Review by tenmozes — APPROVED** [review]

File: charts/victoria-metrics-anomaly/values.yaml [hunk 2]
```diff
@@ -121,6 +121,15 @@ remote:
...
       password: ""
+
+    # -- Bearer token to use for authentication
+    bearer_token: ""
+
+    # -- Whether to verify TLS certificates when connecting to read endpoint
+    verify_tls: true
+
+    # -- List of extra filters to apply to all queries. See: https://docs.victoriametrics.com/#prometheus-querying-api-enhancements
+    extra_filters: [ ]
```
Comment (tenmozes) [inline charts/victoria-metrics-anomaly/values.yaml:132]: what about extra labels? for both - reading and writing

Comment (zekker6) [reply charts/victoria-metrics-anomaly/values.yaml:132]: For reading `extra_filters` can already supersede `extra_label` parameter and we haven't seen a reason to add both atm as it wasn't requested.

For writing vmanomaly appends all fields passed in `metric_format`, e.g.:
```
metric_format:
  name: "node_exporter_$VAR"
  query_key: "for"
  model: prophet
```
Here `name` and `query_key` are reserved names, but `model` label will be passed as-is and will result in appending `model=prophet` to all emitted metrics.

**Pull request merged**